code/style.css

```diff
@@ -150,84 +283,187 @@ button {
-  margin-left: 15%;
+.win-or-lose h1 {
+  margin: 71px 0px 0px 0px;
+  font-size: 50px;
 }
 
-.win-or-lose h1 {
-  margin-bottom: 32px;
+.win-or-lose h2 {
+  margin: 27px 0px 9px 0px;
+  font-size: 30px;
+}
+
+.win-or-lose h3 {
+  margin-top: 12px;
+}
+
+.winOrLoseTime {
+  display: flex;
+  justify-content: center;
+}
+
+.winOrLoseTime h3{
+  margin-top: 10px;
+}
+
+#playAgain {
+  margin-top: 58px;
+}
+
+.winOrLoseTime.guess-container {
+  margin: 0 25px;
+}
+
+.card-lose {
+  height: 205px;
+  width: 204px;
+  animation-duration: 2s;
+  animation-iteration-count: infinite;
+  margin: 0 auto 0 auto;
+  transform-origin: bottom;
+  animation-name: bounce;
+  animation-timing-function: ease;
+
+}
+@keyframes bounce {
+    0%   { transform: scale(1,1)    translateY(0); }
+    10%  { transform: scale(1.1,.9) translateY(0); }
+    30%  { transform: scale(.9,1.1) translateY(-100px); }
+    50%  { transform: scale(1,1)    translateY(0); }
+    57%  { transform: scale(1,1)    translateY(-7px); }
+    64%  { transform: scale(1,1)    translateY(0); }
+    100% { transform: scale(1,1)    translateY(0); }
+}
+
+.card-win {
+  height: 252px;
+  width: 204px;
+  border: 15px dotted var(--secondary);
+  animation-name: blinking;
+  animation-duration: 1s;
+  animation-iteration-count: 100;
+  background-color: rgb(255, 248, 241);
+  background-clip: padding-box;
+  margin: 0px 0px 30px 0px;
+}
+
+@keyframes blinking {
+  50% {
+    border-color: red;
+  }
 }
 
 @media (max-width: 768px) {
   body {
     flex-direction: column;
   }
 
+  .header {
+    height: 179px;
+  }
+
   .question-section {
     width: 100%;
     min-height: 0;
   }
   button {
     padding: 8px 16px;
   }
+
   h1 {
     font-size: 24px;
     line-height: 30px;
   }
 
+  .question-section h1 {
+    top: -361px;
+  }
+
   .guess-who-icon {
-    width: 51px;
+    width: 103px;
+    left: -28px;
+  }
+
+  .guess-who-background {
+   width: 66px;
+    top: 157px;
+    left: -26px;
   }
 
   .board-wrapper {
     width: 100%;
   }
 
+  select {
+    border-radius: 30px;
+  }
+
   .game-board {
     width: 100%;
     max-width: 750px;
+    margin: 5px 0px;
+    justify-content: center;
   }
 
   .card {
     width: 113px;
-    position: relative;
+    height: 142px;
   }
 
-  .card .guess {
+  .inner-card {
+    transition: transform 1s;
+  }
+
+  .card-front .guess {
     display: flex;
     top: 0;
-    position: absolute;
-    justify-content: flex-end;
+    position: relative;
+    justify-content: center;
   }
 
-  .card .guess span {
+  .card-front .guess span {
     display: none;
   }
 
-  .card .guess .filled-button {
+  .card-front .guess .filled-button {
     padding: 6px 11px;
     margin-bottom: 1px;
     background: white;
     color: var(--primary);
     border: none;
   }
+
+  .win-or-lose {
+    max-width: 700px;
+    padding: 0 20px;
+  }
+
+  .win-or-lose h1 {
+    line-height: 50px;
+    margin-top: 20px;
+  }
+
+  label {
+    display: none;
+  }
 }
+
```

Comment:
I guess I have to come up with something constructive as well. It's difficult, since I'm so impressed with what you accomplished. The only thing I noticed was some of the alignment as I changed dimensions in Inspector. The arrow overlaps the h1 in some sizes and the gorgeous old guy peeking down gets overlapped by the h1 in other sizes. But these are really minor details.

Reply:
Good point! I will try to resolve this somehow